Notify/handler.ts

```diff
@@ -210,7 +215,23 @@ export const Notify = (
     sendNotification
   );
   const middlewaresWrap = withRequestMiddlewares(
-    RequiredBodyPayloadMiddleware(NotificationInfo)
+    RequiredBodyPayloadMiddleware(NotificationInfo),
+    AzureAllowBodyPayloadMiddleware(
+      t.interface({
+        notification_type: t.literal(NotificationTypeEnum.MESSAGE)
+      }),
+      new Set([UserGroup.ApiNewMessageNotify])
+    ),
+    AzureAllowBodyPayloadMiddleware(
+      t.interface({
+        notification_type: t.union([
+          t.literal(NotificationTypeEnum.REMINDER_PAYMENT),
+          t.literal(NotificationTypeEnum.REMINDER_PAYMENT_LAST),
+          t.literal(NotificationTypeEnum.REMINDER_READ)
+        ])
+      }),
+      new Set([UserGroup.ApiReminderNotify])
```

Comment:
Just an idea, we could extract two codec, `MessageNotificationInfo` and `ReminderNotificationInfo`

Reply:
done, thank you!